### DISCUSSION TITLE
Need to detect if a new version of the app is available. Is this the right lib?

### DISCUSSION THREAD

**Discussion by baptisteArno** [shadowwalker/next-pwa#458] [Q&A]
Hi,

I run a SaaS, and I'd like to display a popup to the user whenever a new deployment of the application is available. It would then just refresh the page.

Is `next-pwa` the right lib, or is it overkill for this?

**Comment by ImBIOS** [comment]
@baptisteArno It seems this repo is dead and the maintainer cannot be contacted for so long now. I've made an immediate response by resurrecting this repo [ImBIOS/next-pwa](https://github.com/ImBIOS/next-pwa).
